templates/quickstart-confluence-master-with-vpc.template.yaml

```diff
@@ -378,12 +389,12 @@ Parameters:
       - db.t2.xlarge
       - db.t2.2xlarge
     ConstraintDescription: Must be a valid RDS instance class from the list.
-    Description: RDS instance type.
+    Description: RDS instance type (must be r4 family if using Aurora)
```

Comment:
Investigate t3 instance type

Reply:
Not gonna follow up on this for now. Aurora template does not support any instance class outside of r4 and not going to bother with a PR at this stage. Will put in backlog.